gallery/visualtests/nxBreadcrumb.js

```diff
@@ -36,6 +36,7 @@ describe('NxBreadcrumb', function() {
     );
 
     await dropdownBtn.click();
+    await wait(TOOLTIP_WAIT);
```

Comment:
I was getting diffs where tooltips had sometimes appeared or partially appeared.  By waiting we ensure they're always there